Adding Become a Mentor Test case

gagannani opened the pull request:
request to become a mentor,
approve to become a mentor,
verification after becoming a mentor,

gagannani committed:
Adding Become a Mentor Test case

request to become a mentor,
approve to become a mentor,
verification after becoming a mentor,

infa-rvarahag reviewed (changes requested):
Page object is not relevant.
1. Page object should be WelcomePage 
This welcome page behaviors  differently  for logged role .
This Welcome page should have all actions for the left Tab buttons 
When others add testcases this page will enhanced as per their  testcases

gagannani committed:
Added update profile TC

added
AppWelcomePage
AppWorkspacePage
BecomeMentorPage
AppProfilePage
TestBecomeAMentor

gagannani committed:
Added update profile TC

added
AppWelcomePage
AppWorkspacePage
BecomeMentorPage
AppProfilePage
TestBecomeAMentor

gagannani committed:
removed stashed code

gagannani committed:
chaged updateProfilePage to updateProfile method

infa-rvarahag committed:
Merge branch 'main' into GaganBranch
